src/Native/System.Native/pal_networking.cpp

```diff
@@ -21,16 +21,14 @@ static_assert(PAL_TRY_AGAIN == TRY_AGAIN, "");
 static_assert(PAL_NO_RECOVERY == NO_RECOVERY, "");
 static_assert(PAL_NO_DATA == NO_DATA, "");
 static_assert(PAL_NO_ADDRESS == NO_ADDRESS, "");
-static_assert(sizeof(uint8_t) == sizeof(char), ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
+static_assert(sizeof(uint8_t) == sizeof(char),
+              ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
```

Comment:
Why did this move to its own line, because of the comment?  Not a big deal, but seems like this actually makes the code more difficult to read because this line ends up differing from all the ones before it.